pkg/controllers/provisioning/provisioner.go

```diff
@@ -189,6 +209,39 @@ func (p *Provisioner) bind(ctx context.Context, node *v1.Node, pods []*v1.Pod) (
 	return nil
 }
 
+// aggregate takes a list of nodes and recombines them into sets of nodes that can be launched simultaneously
+// with the same set of cloud provider create parameters
+func (p *Provisioner) aggregate(nodes []*scheduling.Node) ([][]*scheduling.Node, error) {
```

Comment:
Isn't this semantically:
```suggestion
func (p *Provisioner) binpack(nodes []*scheduling.Node) ([][]*scheduling.Node, error) {
```